examples/multimodal-modeling/hf/main.py

```diff
@@ -112,10 +112,13 @@ def setup_args():
                         help="To determine whether the quantization should handle vision component.")
     
     # ========== Calibration Datasets ============= 
-    parser.add_argument("--image_path", default="coco", type=str,
-                        help="The dataset for quantization training. It can be a custom one.")
+    parser.add_argument("--extra_data_dir", default="", type=str,
+                        help="Dataset dir for storing images/audio/videos. "
+                        "Can be a dir path or multiple dir path with format as "
+                        "'image=path_to_image,video=path_to_video,audio=path_to_audio'"
+                        "By default, it will search in the relative path.")
     
-    parser.add_argument("--question_path", default=None, type=str,
+    parser.add_argument("--dataset", default=None, type=str,
                             help="The dataset for quantization training. It can be a custom one.")
```

Comment:
if __main__  could support all the functions in the example, we could delete the example